src/GraphQLParser.Tests/Visitors/SDLPrinterTests.cs

```diff
@@ -580,6 +580,8 @@ type Query
     [InlineData(28, "Test \\n escaping", "Test \\\\n escaping", false)]
     [InlineData(29, "Test \\u1234 escaping", "Test \\\\u1234 escaping", false)]
     [InlineData(30, "Test \\ escaping", "Test \\\\ escaping", false)]
+    [InlineData(31, "t\"est\n  line2", "\nt\"est\nline2\n", true)] // BlockString with double quote inside
+    [InlineData(32, "t\\\"\"\"est\n  line2", "\nt\\\"\"\"est\nline2\n", true)] // BlockString with triple double quote inside
     public async Task SDLPrinter_Should_Print_BlockStrings(int number, string input, string expected, bool isBlockString)
```

Comment:
These new test conditions look good.

We cannot test for a string with a CR in it because all strings here are interpreted as a blockstring.  Since a blockstring cannot contain a CR (it would be parsed as a LF), it is impossible to test here what happens when the code tries to serialize a string containing a CR.  We would need to add it to another test.  We also cannot test for beginning or trailing whitespace and so on.  All that we can test for here is strings that can be encoded as blockstrings to begin with.